ex2_hal/adcs/equipment_handler/inc/adcs_handler.h

```diff
@@ -514,7 +523,7 @@ ADCS_returnState ADCS_finalize_upload_block(uint8_t file_dest, uint32_t offset,
 ADCS_returnState ADCS_reset_upload_block(void);
 ADCS_returnState ADCS_reset_file_list_read_pointer(void);
 ADCS_returnState ADCS_initiate_download_burst(uint8_t msg_length, bool ignore_hole_map);
-ADCS_returnState ADCS_receive_download_burst(uint8_t *hole_map, int32_t file_des, uint16_t length_bytes);
+static ADCS_returnState ADCS_receive_download_burst(uint8_t *hole_map, int32_t file_des, uint16_t length_bytes);
```

Comment:
Static function in header file is not allowed

Reply:
Ah thanks for catching that